app/controllers/api/v3/provider_suggestions_controller.rb

```diff
@@ -5,7 +5,6 @@ class ProviderSuggestionsController < API::V3::ApplicationController
 
       def index
         return render(status: :bad_request) if params[:query].nil? || params[:query].length < 3
-        return render(status: :bad_request) unless begins_with_alphanumeric(params[:query])
```

Comment:
As per my other comment there's no test showing the new behaviour. We need a test to prevent future regressions.

Reply:
There is no 'new' behaviour being added. Rather I am removing behaviour that was added in error. We are just reverting back to the default behaviour, hence why no test required in my opinion. What do other's think?

Reply:
Personally I'd add a test to ensure that we won't get a regression in future for *any* reason including rails and library (gem) upgrades. The test would be living documentation that we expect this behaviour to work. I can see that it's debatable, but that's what I'd do.